### ISSUE TITLE
Need date created from API for articles

### ISSUE THREAD

**Issue by purduekenny** [PurdueAgComm/Extension-2018#1]
$article->datCreated doesn't exist in the current object, could we get that added? 

Located in
``src/partials/feed-state-articles.php``

**Comment by sfp-john** [comment]
I've added in datCreated and datModified (datModified seems to be the correct date to use here consistently?)

I have some concerns about the number of API calls taking place now on the article feed list pages, for each article result in the paginated results, we are now making two additional API calls to get the images as well as the article details (for dates).  For example, not including the bootstrap and header/footer elements, an article list of 10 results, will now be 21 unique API calls.  Once this project is near completion, let's discuss caching plans to help avoid excessive use of the API and keep performance reasonable.

https://github.com/PurdueAgComm/Extension-2018/commit/99f725367d5e6a3f42424fc02689568a7fd4ab7b

**Comment by sfp-nathan** [comment]
It looks like the main issue here has been resolved.  For now, I'm going to be focusing on some of the more critical issues open, and if there is time, I'll look into caching.

**Comment by purduekenny** [comment]
Yeah this issue was just kept open because of the caching opportunity.
